geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/classpath/ClassPathUtils.scala

```diff
@@ -136,4 +136,20 @@ object ClassPathUtils extends LazyLogging {
   def cleanClassPathURL(url: String): String =
     URLDecoder.decode(url, "UTF-8").replace("file:", "").replace("!", "")
 
+  def loadClassPathFromEnv(prop: String): Seq[File] = {
```

Comment:
I am attempting to replicate the HADOOP_CLASSPATHS code...which I think is:
1. colon (```:```) separated entries
2. if it ends with ```/*``` then expand out and fetch just jars...seems that java operates that way
3. take specific files if listed.
4. subdirs are not recursively searched but all files in a dir are fetched (not just jars)